core/src/main/kotlin/io/github/chrislo27/toolboks/util/gdxutils/InputUtils.kt

```diff
@@ -44,7 +45,7 @@ fun Input.isButtonJustPressed(button: Int): Boolean {
 }
 
 fun Input.isControlDown(): Boolean {
-    return isKeyPressed(Input.Keys.CONTROL_LEFT) || isKeyPressed(Input.Keys.CONTROL_RIGHT)
+    return isKeyPressed(Input.Keys.CONTROL_LEFT) || isKeyPressed(Input.Keys.CONTROL_RIGHT) || isMac && isKeyPressed(Input.Keys.SYM)
```

Comment:
I disagree with the idea of adding the ⌘ key as a replacement for CTRL. All control-scheme text is already shown as CTRL.